src/pages/library/common.js

```diff
@@ -48,35 +48,22 @@ const groupByFeaturedTags = (workspaces, sidebarSections) => _.flow(
   _.fromPairs
 )(sidebarSections)
 
-const Sidebar = ({ onSectionFilter, onTagFilter, sections, selectedSections, selectedTags, listDataByTag }) => {
-  const unionSectionWorkspacesCount = ({ tags }) => _.flow(
-    _.flatMap(tag => listDataByTag[tag]),
-    _.uniq,
-    _.size
-  )(tags)
-
+const Sidebar = ({ onTagFilter, sections, selectedTags, listDataByTag }) => {
```

Comment:
There might be some missing functionality, do we have to get rid of `onSectionFilter`, `selectedSections`? I think that might be used somewhere

Reply:
I can add them back, but they appear to only be used in the `keepCollapsed` code block, and I couldn't find any uses of `keepCollapsed` in the code. If it's really unused I could should also remove the places where `onSectionFilter` and `selectedSections` is passed in.

Can we be sure something is unused by looking at the code? If not, is there a manual test I can do to see if I've broken something?

Reply:
+1 to removing if unused; if the param doesn't show up in any invocations of the component, it's unused.

Reply:
It turns out it was removed by accident. Adding it back wasn't easy because (of course) the code that was temporarily unreachable no longer works. I (or @petesantos ) will make another PR to add `keepCollapsed` back in.